framework/captcha/drivers/DriverFactory.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace yii\captcha\drivers;
+
+use Yii;
+use yii\base\Object;
+use yii\base\InvalidConfigException;
+
+/**
+ *
+ */
+class DriverFactory extends Object
```

Comment:
Independent instantiation of it at different classes: `Captcha` and `CaptchaAction` is inconsistent: it should either belong to single class only - or be and application component.
I do not consider arguments like 'you can configure this class via DI' cause DI usage should not be used for common configuration.